### DISCUSSION TITLE
Use frameworks or not

### DISCUSSION THREAD

**Discussion by iosifKakalelis** [community/community#33045] [GitHub Education] [answered]
@NBISweden/sysdev-recruitment
Hej!
Do you want to replace the current implementation with component based frameworks or that is not part of the task?
https://github.com/NBISweden/task-for-frontend-2022-joseph13g

Kind Regards,
Iosif Kakalelis

**Comment by jfullstackdev** [comment]
seems this post is not intended here ? or is it ?

**Reply by iosifKakalelis** [reply]
Hello, It is hard to stay. It is the first time I am using this feature

**Comment by airtower-luna** [comment, marked as answer]
This is the GitHub Community, for general (and public!) discussion mostly about Git and GitHub. Your question looks more like it's meant as a mail or chat message to a specific person or maybe company. I hope you can find a way to actually reach them. :sweat_smile:
